app/sitemap.xml

```diff
@@ -1,14 +1,15 @@
 ---
-layout:
+layout: null
 sitemap: false
 ---
 <?xml version="1.0" encoding="utf-8"?>
 <urlset xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.sitemaps.org/schemas/sitemap/0.9 http://www.sitemaps.org/schemas/sitemap/0.9/sitemap.xsd" xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
-{% for page in site.pages %}{% if page.sitemap != null and page.sitemap != false %}
+{% for page in site.pages %}{% if page.sitemap != false %}
   <url>
     <loc>{{ site.url }}{{ page.url }}</loc>
     <lastmod>{{ site.time | date_to_xmlschema}}</lastmod>
     <changefreq>weekly</changefreq>
+    <priority>1.0</priority>
```

Comment:
correct, I added it because I wanted to add a lower priority for older doc versions, didn't get the logic working quite right yet ... but added the priority tag for now.